src/confluent_kafka/admin/__init__.py

```diff
@@ -1205,3 +1221,40 @@ def list_offsets(self, topic_partition_offsets, **kwargs):
 
         super(AdminClient, self).list_offsets(topic_partition_offsets_list, f, **kwargs)
         return futmap
+
+    def delete_records(self, topic_partition_offsets, **kwargs):
+        """
+        Deletes all the records before the specified offsets (not including),
+        in the specified Topic and Partitions.
```

Comment:
Written this way they seem like classes
```suggestion
        in the specified topics and partitions.
```